beetsplug/replaygain.py

```diff
@@ -138,6 +144,17 @@ def compute_album_gain(self, album):
             raise ReplayGainError(u'no output from bs1770gain')
         return AlbumGain(output[-1], output[:-1])
 
+    def compute_track_r128_gain(self, items):
+        """Computes the track gain of the given tracks, returns a list
+        of TrackGain objects.
+        """
+
+        method = self.method
+        self.method = '--ebu'
+        output = self.compute_gain(items, False)
+        self.method = method
```

Comment:
Hmm, yes—mutating and restoring the object's state is a bit hacky. Maybe this decision should be made inside `compute_gain`? The idea would be to write a clear policy about when the "format-specific default" overrides the global default or the user's configuration.